main.go

```diff
@@ -32,6 +34,7 @@ func main() {
 	flag.BoolVar(&noshading, "noshading", false, "do not add shading, this will make unicorns look flatter")
 	flag.BoolVar(&nograss, "nograss", false, "do not add grass to the ground")
 	flag.BoolVar(&serial, "serial", false, "do not parallelize the drawing")
+	flag.BoolVar(&writeStdout, "stdout", false, "write image to STDOUT")
```

Comment:
If we change the info message (see my other comment) to go to stderr instead of stdout, then it seems like this option isn't necessary anymore, since you can specify stdout as the outfile.

At least on my machine, this works fine:

```
$ ./go-unicornify -r -o /dev/stdout
```

What do you think?